pkg/asset/installconfig/nutanix/validation.go

```diff
@@ -0,0 +1,63 @@
+package nutanix
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/pkg/errors"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+
+	"github.com/openshift/installer/pkg/types"
+	nutanixtypes "github.com/openshift/installer/pkg/types/nutanix"
+	"github.com/openshift/installer/pkg/types/nutanix/validation"
+)
+
+// Validate executes platform-specific validation.
+func Validate(ic *types.InstallConfig) error {
+	if ic.Platform.Nutanix == nil {
+		return errors.New(field.Required(field.NewPath("platform", "nutanix"), "Nutanix validation requires a Nutanix platform configuration").Error())
+	}
+
+	p := ic.Platform.Nutanix
+	if errs := validation.ValidatePlatform(p, field.NewPath("platform").Child("nutanix")); len(errs) != 0 {
+		return errs.ToAggregate()
+	}
+
+	_, err := nutanixtypes.CreateNutanixClient(context.TODO(), p.PrismCentral, p.Port, p.Username, p.Password)
+	if err != nil {
+		return errors.New(field.InternalError(field.NewPath("platform", "nutanix"), errors.Wrapf(err, "unable to connect to Prism Central %s.", p.PrismCentral)).Error())
+	}
+
+	return nil
+}
+
+// ValidateForProvisioning performs platform validation specifically for installer-
+// provisioned infrastructure. In this case, self-hosted networking is a requirement
+// when the installer creates infrastructure for nutanix clusters.
+func ValidateForProvisioning(ic *types.InstallConfig) error {
+	if ic.Platform.Nutanix == nil {
+		return errors.New(field.Required(field.NewPath("platform", "nutanix"), "Nutanix validation requires a Nutanix platform configuration").Error())
```

Comment:
the platform is nutanix rather than Nutanix. Recommended change:
nutanix validation requires a nutanix platform configuration